### PR TITLE
Update EQ maintenance page generic

### PR DESCRIPTION
**Pull request by Yuyuutsu**
### What is the context of this PR?
The current maintenance page is business specific. Since we will be supporting both Business/Social, we will need to make the maintenance page generic as we have with runner.
Changes made:
- Update the title from `ONS Business Surveys` to `ONS Surveys`
- Remove the cookie banner.
- Remove the Contact us link in footer
- Remove the Cookies link in footer
- Update the DS version to the latest in the source HTML (currently 58.0.2). 

### How to review 
Build docker image and run container. 
Check if all changes are made

### CODE REVIEW

**Comment by Yuyuutsu** [comment]
[Make EQ maintenance page generic (1 SP)](https://trello.com/c/v8AafQtJ/5434-make-eq-maintenance-page-generic-1-sp)

File: static-html/service.html [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
-        <link rel="manifest" href="https://cdn.ons.gov.uk/sdc/design-system/44.0.2/favicons/main/en/manifest.json" />
+        <meta name="msapplication-config" content="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/browserconfig.json" />
+        <link rel="icon" type="image/x-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2favicons/main/en/favicon.ico" />
+        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/favicon-32x32.png" sizes="32x32" />
+        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/favicon-16x16.png" sizes="16x16" />
+        <link rel="mask-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/safari-pinned-tab.svg" color="#000000" />
+        <link rel="apple-touch-icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/apple-touch-icon.png" sizes="180x180" />
+        <link rel="manifest" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/main/en/manifest.json" />
```
Comment (rmccar) [inline static-html/service.html]:
```suggestion
        <meta name="msapplication-config" content="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/browserconfig.json" />
        <link rel="icon" type="image/x-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/favicon.ico" />
        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/favicon-32x32.png" sizes="32x32" />
        <link rel="icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/favicon-16x16.png" sizes="16x16" />
        <link rel="mask-icon" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/safari-pinned-tab.svg" color="#000000" />
        <link rel="apple-touch-icon" type="image/png" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/apple-touch-icon.png" sizes="180x180" />
        <link rel="manifest" href="https://cdn.ons.gov.uk/sdc/design-system/58.0.2/favicons/manifest.json" />
```

These links need updating

**Review by rmccar — COMMENTED** [review]
Other than these link changes it looks good

File: static-html/service.html (hunk 1 above)
Comment (Yuyuutsu) [reply static-html/service.html]: Tested and re run. All works, thanks !

**Review by rmccar — APPROVED** [review]

**Review by petechd — COMMENTED** [review]
Maybe we should reformat using djlint since we are here?
List of issues djlint found before reformat:

**H030 2:0 Consider adding a meta description. <html lang="en"> 
H031 2:0 Consider adding meta keywords. <html lang="en"> 
H029 29:16 Consider using lowercase form method values. <form method="POST"
H017 40:52 Tag should be self closing. <path d="M0,34.6c.8-
H017 43:52 Tag should be self closing. <path d="M53.06,6.42
H017 46:52 Tag should be self closing. <path d="M82,47.49c-
H017 47:52 Tag should be self closing. <path d="M106.36,23.
H017 48:52 Tag should be self closing. <path d="M121.53,23.
H017 49:52 Tag should be self closing. <path d="M132.85,16.
H017 50:52 Tag should be self closing. <path d="M150.53,47.
H017 51:52 Tag should be self closing. <path d="M162.84,35.
H017 52:52 Tag should be self closing. <path d="M198.57,23.
H017 53:52 Tag should be self closing. <path d="M217.28,47.
H017 54:52 Tag should be self closing. <path d="M244.08,23.
H017 55:52 Tag should be self closing. <path d="M277.42,47.
H017 56:52 Tag should be self closing. <path d="M297.52,47.
H017 57:52 Tag should be self closing. <path d="M322,47.69c
H017 58:52 Tag should be self closing. <path d="M331.91,17.
H017 59:52 Tag should be self closing. <path d="M350.88,47.
H017 60:52 Tag should be self closing. <path d="M382.52,47.
H017 61:52 Tag should be self closing. <path d="M403.18,47.
H017 63:52 Tag should be self closing. <path d="M445.39,47.
H017 64:52 Tag should be self closing. <path d="M470.41,47.
H017 65:52 Tag should be self closing. <path d="M487.27,47.
H017 66:52 Tag should be self closing. <path d="M511.73,47.
H017 67:52 Tag should be self closing. <path d="M521.66,17.
H017 68:52 Tag should be self closing. <path d="M536.19,47.
H017 69:52 Tag should be self closing. <path d="M559.83,47.
H017 70:52 Tag should be self closing. <path d="M569.77,17.
H017 71:52 Tag should be self closing. <path d="M588.14,47.
H017 72:52 Tag should be self closing. <path d="M605.1,47.7
H017 220:36 Tag should be self closing. <path 
H023 232:36 Do not use entity references. &copy;**

**Comment by Yuyuutsu** [comment]
Yeah sounds good

**Comment by rmccar** [comment]
> Maybe we should reformat using djlint since we are here? List of issues djlint found before reformat:
> 
> **H030 2:0 Consider adding a meta description. H031 2:0 Consider adding meta keywords. H029 29:16 Consider using lowercase form method values. <form method="POST" H017 40:52 Tag should be self closing. <path d="M0,34.6c.8- H017 43:52 Tag should be self closing. <path d="M53.06,6.42 H017 46:52 Tag should be self closing. <path d="M82,47.49c- H017 47:52 Tag should be self closing. <path d="M106.36,23. H017 48:52 Tag should be self closing. <path d="M121.53,23. H017 49:52 Tag should be self closing. <path d="M132.85,16. H017 50:52 Tag should be self closing. <path d="M150.53,47. H017 51:52 Tag should be self closing. <path d="M162.84,35. H017 52:52 Tag should be self closing. <path d="M198.57,23. H017 53:52 Tag should be self closing. <path d="M217.28,47. H017 54:52 Tag should be self closing. <path d="M244.08,23. H017 55:52 Tag should be self closing. <path d="M277.42,47. H017 56:52 Tag should be self closing. <path d="M297.52,47. H017 57:52 Tag should be self closing. <path d="M322,47.69c H017 58:52 Tag should be self closing. <path d="M331.91,17. H017 59:52 Tag should be self closing. <path d="M350.88,47. H017 60:52 Tag should be self closing. <path d="M382.52,47. H017 61:52 Tag should be self closing. <path d="M403.18,47. H017 63:52 Tag should be self closing. <path d="M445.39,47. H017 64:52 Tag should be self closing. <path d="M470.41,47. H017 65:52 Tag should be self closing. <path d="M487.27,47. H017 66:52 Tag should be self closing. <path d="M511.73,47. H017 67:52 Tag should be self closing. <path d="M521.66,17. H017 68:52 Tag should be self closing. <path d="M536.19,47. H017 69:52 Tag should be self closing. <path d="M559.83,47. H017 70:52 Tag should be self closing. <path d="M569.77,17. H017 71:52 Tag should be self closing. <path d="M588.14,47. H017 72:52 Tag should be self closing. <path d="M605.1,47.7 H017 220:36 Tag should be self closing. <path H023 232:36 Do not use entity references. ©**

This is another thing that would be fixed if we started using the DS macros for this repo. Maybe its something we should look into doing?

**Comment by Yuyuutsu** [comment]
> > Maybe we should reformat using djlint since we are here? List of issues djlint found before reformat:
> > **H030 2:0 Consider adding a meta description. H031 2:0 Consider adding meta keywords. H029 29:16 Consider using lowercase form method values. <form method="POST" H017 40:52 Tag should be self closing. <path d="M0,34.6c.8- H017 43:52 Tag should be self closing. <path d="M53.06,6.42 H017 46:52 Tag should be self closing. <path d="M82,47.49c- H017 47:52 Tag should be self closing. <path d="M106.36,23. H017 48:52 Tag should be self closing. <path d="M121.53,23. H017 49:52 Tag should be self closing. <path d="M132.85,16. H017 50:52 Tag should be self closing. <path d="M150.53,47. H017 51:52 Tag should be self closing. <path d="M162.84,35. H017 52:52 Tag should be self closing. <path d="M198.57,23. H017 53:52 Tag should be self closing. <path d="M217.28,47. H017 54:52 Tag should be self closing. <path d="M244.08,23. H017 55:52 Tag should be self closing. <path d="M277.42,47. H017 56:52 Tag should be self closing. <path d="M297.52,47. H017 57:52 Tag should be self closing. <path d="M322,47.69c H017 58:52 Tag should be self closing. <path d="M331.91,17. H017 59:52 Tag should be self closing. <path d="M350.88,47. H017 60:52 Tag should be self closing. <path d="M382.52,47. H017 61:52 Tag should be self closing. <path d="M403.18,47. H017 63:52 Tag should be self closing. <path d="M445.39,47. H017 64:52 Tag should be self closing. <path d="M470.41,47. H017 65:52 Tag should be self closing. <path d="M487.27,47. H017 66:52 Tag should be self closing. <path d="M511.73,47. H017 67:52 Tag should be self closing. <path d="M521.66,17. H017 68:52 Tag should be self closing. <path d="M536.19,47. H017 69:52 Tag should be self closing. <path d="M559.83,47. H017 70:52 Tag should be self closing. <path d="M569.77,17. H017 71:52 Tag should be self closing. <path d="M588.14,47. H017 72:52 Tag should be self closing. <path d="M605.1,47.7 H017 220:36 Tag should be self closing. <path H023 232:36 Do not use entity references. ©**
> 
> This is another thing that would be fixed if we started using the DS macros for this repo. Maybe its something we should look into doing?

Yeah sounds like a good idea. Would it be out of scope for this ticket ?

**Comment by Yuyuutsu** [comment]
> Maybe we should reformat using djlint since we are here? List of issues djlint found before reformat:
> 
> **H030 2:0 Consider adding a meta description. H031 2:0 Consider adding meta keywords. H029 29:16 Consider using lowercase form method values. <form method="POST" H017 40:52 Tag should be self closing. <path d="M0,34.6c.8- H017 43:52 Tag should be self closing. <path d="M53.06,6.42 H017 46:52 Tag should be self closing. <path d="M82,47.49c- H017 47:52 Tag should be self closing. <path d="M106.36,23. H017 48:52 Tag should be self closing. <path d="M121.53,23. H017 49:52 Tag should be self closing. <path d="M132.85,16. H017 50:52 Tag should be self closing. <path d="M150.53,47. H017 51:52 Tag should be self closing. <path d="M162.84,35. H017 52:52 Tag should be self closing. <path d="M198.57,23. H017 53:52 Tag should be self closing. <path d="M217.28,47. H017 54:52 Tag should be self closing. <path d="M244.08,23. H017 55:52 Tag should be self closing. <path d="M277.42,47. H017 56:52 Tag should be self closing. <path d="M297.52,47. H017 57:52 Tag should be self closing. <path d="M322,47.69c H017 58:52 Tag should be self closing. <path d="M331.91,17. H017 59:52 Tag should be self closing. <path d="M350.88,47. H017 60:52 Tag should be self closing. <path d="M382.52,47. H017 61:52 Tag should be self closing. <path d="M403.18,47. H017 63:52 Tag should be self closing. <path d="M445.39,47. H017 64:52 Tag should be self closing. <path d="M470.41,47. H017 65:52 Tag should be self closing. <path d="M487.27,47. H017 66:52 Tag should be self closing. <path d="M511.73,47. H017 67:52 Tag should be self closing. <path d="M521.66,17. H017 68:52 Tag should be self closing. <path d="M536.19,47. H017 69:52 Tag should be self closing. <path d="M559.83,47. H017 70:52 Tag should be self closing. <path d="M569.77,17. H017 71:52 Tag should be self closing. <path d="M588.14,47. H017 72:52 Tag should be self closing. <path d="M605.1,47.7 H017 220:36 Tag should be self closing. <path H023 232:36 Do not use entity references. ©**

Made the changes using djlint. Most of the linting issues I had to fix manually because djlint was not reformatting the linting errors. 
There are two linting issues left but seems unnecessary to resolve.
```
H030 2:0 Consider adding a meta description. <html lang="en"> 
H031 2:0 Consider adding meta keywords. <html lang="en"> 
```

**Comment by rmccar** [comment]
> > Maybe we should reformat using djlint since we are here? List of issues djlint found before reformat:
> > **H030 2:0 Consider adding a meta description. H031 2:0 Consider adding meta keywords. H029 29:16 Consider using lowercase form method values. <form method="POST" H017 40:52 Tag should be self closing. <path d="M0,34.6c.8- H017 43:52 Tag should be self closing. <path d="M53.06,6.42 H017 46:52 Tag should be self closing. <path d="M82,47.49c- H017 47:52 Tag should be self closing. <path d="M106.36,23. H017 48:52 Tag should be self closing. <path d="M121.53,23. H017 49:52 Tag should be self closing. <path d="M132.85,16. H017 50:52 Tag should be self closing. <path d="M150.53,47. H017 51:52 Tag should be self closing. <path d="M162.84,35. H017 52:52 Tag should be self closing. <path d="M198.57,23. H017 53:52 Tag should be self closing. <path d="M217.28,47. H017 54:52 Tag should be self closing. <path d="M244.08,23. H017 55:52 Tag should be self closing. <path d="M277.42,47. H017 56:52 Tag should be self closing. <path d="M297.52,47. H017 57:52 Tag should be self closing. <path d="M322,47.69c H017 58:52 Tag should be self closing. <path d="M331.91,17. H017 59:52 Tag should be self closing. <path d="M350.88,47. H017 60:52 Tag should be self closing. <path d="M382.52,47. H017 61:52 Tag should be self closing. <path d="M403.18,47. H017 63:52 Tag should be self closing. <path d="M445.39,47. H017 64:52 Tag should be self closing. <path d="M470.41,47. H017 65:52 Tag should be self closing. <path d="M487.27,47. H017 66:52 Tag should be self closing. <path d="M511.73,47. H017 67:52 Tag should be self closing. <path d="M521.66,17. H017 68:52 Tag should be self closing. <path d="M536.19,47. H017 69:52 Tag should be self closing. <path d="M559.83,47. H017 70:52 Tag should be self closing. <path d="M569.77,17. H017 71:52 Tag should be self closing. <path d="M588.14,47. H017 72:52 Tag should be self closing. <path d="M605.1,47.7 H017 220:36 Tag should be self closing. <path H023 232:36 Do not use entity references. ©**
> 
> Made the changes using djlint. Most of the linting issues I had to fix manually because djlint was not reformatting the linting errors. There are two linting issues left but seems unnecessary to resolve.
> 
> ```
> H030 2:0 Consider adding a meta description. <html lang="en"> 
> H031 2:0 Consider adding meta keywords. <html lang="en"> 
> ```

Those are more for SEO and I don't think are currently set in runner so maybe there is some config that can be set in djlint to turn those particular errors off?

**Comment by rmccar** [comment]
> > > Maybe we should reformat using djlint since we are here? List of issues djlint found before reformat:
> > > **H030 2:0 Consider adding a meta description. H031 2:0 Consider adding meta keywords. H029 29:16 Consider using lowercase form method values. <form method="POST" H017 40:52 Tag should be self closing. <path d="M0,34.6c.8- H017 43:52 Tag should be self closing. <path d="M53.06,6.42 H017 46:52 Tag should be self closing. <path d="M82,47.49c- H017 47:52 Tag should be self closing. <path d="M106.36,23. H017 48:52 Tag should be self closing. <path d="M121.53,23. H017 49:52 Tag should be self closing. <path d="M132.85,16. H017 50:52 Tag should be self closing. <path d="M150.53,47. H017 51:52 Tag should be self closing. <path d="M162.84,35. H017 52:52 Tag should be self closing. <path d="M198.57,23. H017 53:52 Tag should be self closing. <path d="M217.28,47. H017 54:52 Tag should be self closing. <path d="M244.08,23. H017 55:52 Tag should be self closing. <path d="M277.42,47. H017 56:52 Tag should be self closing. <path d="M297.52,47. H017 57:52 Tag should be self closing. <path d="M322,47.69c H017 58:52 Tag should be self closing. <path d="M331.91,17. H017 59:52 Tag should be self closing. <path d="M350.88,47. H017 60:52 Tag should be self closing. <path d="M382.52,47. H017 61:52 Tag should be self closing. <path d="M403.18,47. H017 63:52 Tag should be self closing. <path d="M445.39,47. H017 64:52 Tag should be self closing. <path d="M470.41,47. H017 65:52 Tag should be self closing. <path d="M487.27,47. H017 66:52 Tag should be self closing. <path d="M511.73,47. H017 67:52 Tag should be self closing. <path d="M521.66,17. H017 68:52 Tag should be self closing. <path d="M536.19,47. H017 69:52 Tag should be self closing. <path d="M559.83,47. H017 70:52 Tag should be self closing. <path d="M569.77,17. H017 71:52 Tag should be self closing. <path d="M588.14,47. H017 72:52 Tag should be self closing. <path d="M605.1,47.7 H017 220:36 Tag should be self closing. <path H023 232:36 Do not use entity references. ©**
> > 
> > 
> > This is another thing that would be fixed if we started using the DS macros for this repo. Maybe its something we should look into doing?
> 
> Yeah sounds like a good idea. Would it be out of scope for this ticket ?

Yeah probably but might be something to bring up in a tech session

**Comment by petechd** [comment]
I would still add that language meta, just because it's a standard thing.

**Comment by Yuyuutsu** [comment]
> I would still add that language meta, just because it's a standard thing.

Done 👍

**Review by petechd — APPROVED** [review]

**Pull request merged**